pkgs/development/libraries/belcard/default.nix

```diff
@@ -7,7 +7,7 @@
 
 stdenv.mkDerivation rec {
   pname = "belcard";
-  version = "5.0.55";
+  version = "5.0.66";
```

Comment:
Shouldn't there be a mtching hash bump?

Reply:
Both tags are on the same commit, so no hash bumps. This is more of a semantic change.